wdl/model/draft2/src/test/scala/wdl/SyntaxHighlightSpec.scala

```diff
@@ -154,7 +157,7 @@ class SyntaxHighlightSpec extends AnyWordSpec with CromwellTimeoutSpec with Matc
         |  \u001b[38;5;33mMap[String, Array[String]]\u001b[0m \u001b[38;5;112mmetadata\u001b[0m
         |  \u001b[38;5;214mscatter\u001b[0m (readgroup in readgroup_list) {
         |    \u001b[38;5;214mcall\u001b[0m \u001b[38;5;253mPairedFastQsToUnmappedBAM\u001b[0m {
-        |      input: library_name=metadata[readgroup][1], run_date=metadata[readgroup][3], readgroup_name=readgroup, platform_name=metadata[readgroup][4], platform_unit=metadata[readgroup][2], fastq_1=fastq_pairs[readgroup][0], fastq_2=fastq_pairs[readgroup][1], sample_name=metadata[readgroup][0], sequencing_center=metadata[readgroup][5]
+        |      input: library_name=metadata[readgroup][1], platform_unit=metadata[readgroup][2], fastq_1=fastq_pairs[readgroup][0], fastq_2=fastq_pairs[readgroup][1], sample_name=metadata[readgroup][0], sequencing_center=metadata[readgroup][5], run_date=metadata[readgroup][3], readgroup_name=readgroup, platform_name=metadata[readgroup][4]
```

Comment:
it appears the ordering of these keys (and hence values) changed between 2.12 and 2.13